### PR TITLE
chore: Added new project #1978

### PR DESCRIPTION
**Pull request by soubhik-111**
## Related Issue
Added a new project name Kaun Banega Crorepati

## Description
Kaun Banega Crorepati (KBC) Frontend Project is an interactive web application inspired by the popular Indian television quiz show, "Kaun Banega Crorepati," where contestants answer multiple-choice questions to win cash prizes. This frontend project is built using HTML, CSS, and JavaScript to create a user-friendly and engaging experience for users.

## Screenshots
image

### CODE REVIEW

**Comment by soubhik-111** [comment]
Issue Number #1978

File: DB/projects.json [hunk 1]
```diff
@@ -5795,5 +5795,23 @@
...
+    "Instagram": "https://www.instagram.com/codewithcrazy/",
+    "YouTube": "https://www.youtube.com/@codewithcrazy",
+    "Twitter": ""
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/soubhik-111/Kaun_banega_crorepati",
+      "title": "Kaun Banega Crorepati",
+      "description": "Kaun Banega Crorepati Frontend Project is an interactive web app with HTML, CSS, JAVASCRIPT. Features include questions, timers,  scores, and animations for an engaging experience.",
+      "tech": ["HTML", "CSS", "JAVASCRIPT"]
```
Comment (priyankarpal) [inline DB/projects.json:5813]: convert all tech stacks to lowercase 
```suggestion
      "tech": ["html", "css", "javascript"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json [hunk 2]
```diff
@@ -5341,7 +5341,25 @@
...
 },
-  {
+{
+  "github_username": "soubhik-111",  
+  "Social_media": {
+    "gitHub": "https://github.com/soubhik-111",
+    "LinkedIn": "https://www.linkedin.com/in/soubhik-banerjee-3a571b223/",
+    "Instagram": "https://www.instagram.com/codewithcrazy/",
+    "YouTube": "https://www.youtube.com/@codewithcrazy",
+    "Twitter": ""
```
[the remark this answers is not in the document]
Comment (soubhik-111) [reply DB/projects.json:5350]: May I know why we need to delete Instagram Youtube and twitter all three?

[the remark this answers is not in the document]
Comment (priyankarpal) [reply DB/projects.json:5350]: currently we are collecting only Twitter, LinkedIn & GitHub accounts link. You haven't added Twitter link so I requested you to remove the Twitter link. For GitHub we are uisgn your GitHub username to show the GitHub icon @soubhik-111

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**